presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -44,6 +44,14 @@ Array Functions
 
     Remove duplicate values from the array ``x``.
 
+.. function:: array_dupes(array(bigint)) -> array(bigint)
+
+    Returns an array with elements that are duplicated in the ``array``. Each element is repeated only once and may be in any order.
+
```

Comment:
'repeated only once' is confusing. Just say the result is a set of elements that occur more than once in the original array.